install/kubernetes/helm/subcharts/ingress/values.yaml

```diff
@@ -9,6 +9,7 @@ service:
   annotations: {}
   loadBalancerIP: ""
   type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
+  #externalTrafficPolicy: Local #change to Local to preserve source IP or Cluster for default behaviour or leave commented out
```

Comment:
I prefer the following:

```
# Update to Local to preserve source IP or Cluster for default behaviour or leave commented out
externalTrafficPolicy:
```

Reply:
This was already added to gateways in #8941, and I didn't want to provide different text for the two.